### PR TITLE
Adding __eq__ method to KeyJar class

### PR DESCRIPTION
**Pull request by rohe**
Added a method that compares two KeyJar instances for equality.

### CODE REVIEW

**Comment by rohe** [comment]
Why does Travis fail on:
ERROR: InvocationError: '/usr/bin/make check-isort'
??

**Comment by tpazderka** [comment]
Because your imports are incorrectly sorted.

File: src/oic/utils/keyio.py [hunk 1]
```diff
@@ -776,6 +776,36 @@ def get_issuer_keys(self, issuer):
...
+        # number of issuer_keys must be equal
+        if self.issuer_keys.keys() != other.issuer_keys.keys():
+            return False
+
+        for issuer in self.issuer_keys.keys():
+            mine = self.get_issuer_keys(issuer)
+            others = other.get_issuer_keys(issuer)
+            if len(mine) != len(others):
+                return False
+            for a_key in mine:
```
Comment (lwm) [inline src/oic/utils/keyio.py]: I think you could replace this nested for logic as `return set(mine) & set(others)`, no?

https://docs.python.org/3/library/stdtypes.html#set

**Comment by lwm** [comment]
You'll need to clean up your commit log. This should be a single commit.

**Pull request closed without merging**